ketcher-autotests/tests/File-Management/Open-And-Save-Files/PNG-Files/png-files.spec.ts

```diff
@@ -51,4 +51,112 @@ test.describe('Saving in .png files', () => {
       await takeEditorScreenshot(page);
     });
   }
+
+  test(`Verify it is possible to export the simple schema with retrosynthetic arrow to PNG`, async ({
+    page,
+  }) => {
+    /*
+    Test case: #2096
+    Description: Validate that schema with retrosynthetic arrow could be saved to rxn2000 file and loaded back
+    */
+    await openFileAndAddToCanvas(
+      'KET/simple-schema-with-retrosynthetic-arrow.ket',
+      page,
+    );
+
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByRole('option', { name: 'PNG Image' }).click();
+    await takeEditorScreenshot(page);
+  });
+
+  test(`Verify it is possible to export the schema with retrosynthetic, angel arrows and plus to PNG`, async ({
+    page,
+  }) => {
+    /*
+    Test case: #2096
+    Description: Validate that schema with retrosynthetic arrow could be saved to rxn2000 file and loaded back
+    */
+    await openFileAndAddToCanvas(
+      'KET/schema-with-retrosynthetic-angel-arrows-and-plus.ket',
+      page,
+    );
+
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByRole('option', { name: 'PNG Image' }).click();
+    await takeEditorScreenshot(page);
+  });
+
+  test(`Verify it is possible to export the schema with vertical retrosynthetic arrow to PNG`, async ({
+    page,
+  }) => {
+    /*
+    Test case: #2096
+    Description: Validate that schema with retrosynthetic arrow could be saved to rxn2000 file and loaded back
+    */
+    await openFileAndAddToCanvas(
+      'KET/schema-with-vertical-retrosynthetic-arrow.ket',
+      page,
+    );
+
+    await selectTopPanelButton(TopPanelButton.Save, page);
+    await clickOnFileFormatDropdown(page);
+    await page.getByRole('option', { name: 'PNG Image' }).click();
+    await takeEditorScreenshot(page);
+  });
```

Comment:
1. Test description seems to be wrong
2. Replace page.getByRole... with function (it should be no locators in the test)
    OR
3. Better to replace 157, 158 and 159 with local function